apiservers/portlayer/restapi/handlers/scopes_handlers.go

```diff
@@ -30,7 +30,12 @@ import (
 
 // ScopesHandlersImpl is the receiver for all of the storage handler methods
 type ScopesHandlersImpl struct {
-	ctx *network.Context
+}
+
+var ctx *network.Context
+
+func GetNetworkContext() *network.Context {
```

Comment:
I need to share it amongst all the handlers. Is there another way I can do that?